common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessage.java

```diff
@@ -73,26 +113,27 @@
      *
      * Example: "123e4567-e89b-12d3-a456-426655440000"
      */
-    String getMessageInstanceId();
+    public String getMessageInstanceId() {
+        return messageInstanceId;
+    }
 
     /**
      * For support and developers - component that generated the error (can be fully qualified Java package or class name).
```

Comment:
Isn't this all for support and developers? I'd scrap this note.
"Component that generates the error". I'll need to understand what stands for 'component' and 'error' here. Is 'a fully qualified Java package or class name' a component or error?

Reply:
"Component that generates the error" means that the component (for example a class) can throw an error. Qualified java package or class name is a component.

Reply:
Then I'd state it directly "Qualified Java package or class name that generates(ed) the error" 
'generates' or 'generated' dependent on whether the error is to be generated or was generated.